### PR TITLE
DEP- 2678 Commited the Redirect ALL Tax Credit URLS to DAv2

### PR DESCRIPTION
**Pull request by sanjayshukla2205**
# DEP- 2678

## Checklist PR Raiser
 - [x]  I've ensured code coverage has not decreased
 - [ ]  I've dealt with any new compilation warnings
 - [x]  I've ensured the team's coding standards have been met (TBC)?
 - [ ]  If relevant, I've created corresponding app-config-XXX changes?
 - [ ]  If I've created new test data, I've ensured there is no real person's data (see https://confluence.tools.tax.service.gov.uk/display/CD/Test+Data+in+the+Open) 
 - [x]  I've considered the impact my changes have on the UI/Journey tests (please do not break them)
-  [ ]  If relevant, I've added links to associated PRs

## Checklist PR Reviewer
 - [ ]  I've checked to ensure all relevant unit tests have been written
 - [ ]  I've checked to ensure the team's coding standards have been met
 - [ ]  I've executed the acceptance test pack locally to ensure there are no functional regressions (Performance tests where relevant)
 - [ ]  If I merge I will ensure I use squash and merge

### CODE REVIEW

**Comment by sanjayshukla2205** [comment]
Hi All, 
  I would like to update that out of the given three URL's mentioned in the story to redirect them to the -ask-hmrc-online link  ( 
 https://www.tax.service.gov.uk/ask-hmrc/chat/ask-hmrc-online ) only two are the eligible candidates as one of the link is already redirected to the ask-hmrc-online link.
Hence the changes has only been done for the below two links:
https://www.tax.service.gov.uk/ask-hmrc/webchat/tax-credits-enquiries?nuance=ivr
https://www.tax.service.gov.uk/ask-hmrc/ivr/taxcredits

File: app/controllers/IvrController.scala [hunk 1]
```diff
@@ -22,17 +22,20 @@ import play.api.mvc.{Action, AnyContent, MessagesControllerComponents}
 import uk.gov.hmrc.play.bootstrap.frontend.controller.FrontendController
 
 import scala.concurrent.Future
+import views.html.CUIViews.AskHmrcOnlineCUIView
```
Comment (lmugnai) [inline app/controllers/IvrController.scala:25]: @sanjayshukla2205 do we still need this import? If not can we remove it and the injected view

File: app/controllers/WebchatController.scala [hunk 2]
```diff
@@ -22,7 +22,7 @@ import javax.inject.{Inject, Singleton}
 import play.api.mvc.{Action, AnyContent, MessagesControllerComponents, RequestHeader}
 import uk.gov.hmrc.play.bootstrap.frontend.controller.FrontendController
 import views.html.webchat._
-
+import views.html.CUIViews.AskHmrcOnlineCUIView
```
Comment (lmugnai) [inline app/controllers/WebchatController.scala:25]: Same with this one. Is it still needed?

File: test/controllers/WebchatControllerSpec.scala [hunk 3]
```diff
@@ -87,8 +87,11 @@ class WebchatControllerSpec
         val result = controller.taxCredits(ivrFakeRequest)
         val doc = asDocument(contentAsString(result))
 
-        status(result) mustBe OK
-        doc.select("h1").text() mustBe "Tax credits: webchat"
+       //  status(result) mustBe OK
+        // doc.select("h1").text() mustBe "Tax credits: webchat"
```
Comment (lmugnai) [inline test/controllers/WebchatControllerSpec.scala:91]: Can we remove these comments please?

**Review by lmugnai — CHANGES_REQUESTED** [review]

File: app/controllers/IvrController.scala (hunk 1 above)
Comment (sanjayshukla2205) [reply app/controllers/IvrController.scala]: Hi Lorenzo, Yes this is no more required, hence i am removing it.

File: app/controllers/WebchatController.scala (hunk 2 above)
Comment (sanjayshukla2205) [reply app/controllers/WebchatController.scala]: Hi Lorenzo, Yes this is no more required, hence i am removing it.

**Review by lmugnai — APPROVED** [review]

**Review by tonymyers — APPROVED** [review]

**Pull request merged**